scripts/btsbot/btsbot.py

```diff
@@ -68,13 +68,11 @@
 
 _LICENSE = "CC BY 4.0"
 
-_VERSION = "10.0.0"
+_VERSION = "1.0.0"
```

Comment:
Note: It was v10 to match the official release of BTSbot data.

Reply:
yeah, but this refers to the MMU version of the dataset